docs/pages/workflow/overview.mdx

```diff
@@ -201,8 +203,9 @@ The core development loop described in the diagram above is a cycle of four main
   config file.
 
 - <Highlight isBlue>**Write native code or modify native project configuration**</Highlight>: This
-  includes writing native code directly or modifying native code configuration. You either need access to
-  the native code project directories to make these changes, or you can write native code with a [local Expo Module](/modules/get-started/#adding-a-new-module-to-an-existing-application).
+  includes writing native code directly or modifying native code configuration. You either need
+  access to the native code project directories to make these changes, or you can write native code
+  with a [local Expo Module](/modules/get-started/#adding-a-new-module-to-an-existing-application).
```

Comment:
```suggestion
  includes writing native code directly or modifying native code configuration. You either need access to the native code project directories to make these changes, or you can write native code with a [local Expo Module](/modules/get-started/#adding-a-new-module-to-an-existing-application).
```

Reply:
That's how Prettier format these lines because they are indented based on our docs Prettier config.